ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -9919,7 +10203,7 @@ spec:
   query: |
     SELECT 1 FROM registry WHERE (path LIKE 'HKEY_USERS\%\Software\Policies\Microsoft\Windows\Installer\AlwaysInstallElevated' AND data = 0);
```

Comment:
@defensivedepth this item should be updated and tested to these values:

Registry key:
HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\Installer:AlwaysInstal lElevated

GPO:
Computer Configuration\Policies\Administrative Templates\Windows Components\Windows Installer\Always install with elevated privileges